data/gallery/fences.ts

```diff
@@ -0,0 +1,133 @@
+import image1 from "public/gallery/fences/1.jpg";
```

Comment:
For now I configured data for 2 galleries but all photos are already in public folder